charts/external-es-proxy/templates/external-es-proxy-networkpolicy.yaml

```diff
@@ -27,6 +27,7 @@ spec:
         matchLabels:
           tier: airflow
           component: webserver
+    {{- if .Values.global.loggingSidecar.enabled }}
```

Comment:
Same question as above - (Why are we restricting this to triggerer for sidecar only?)